src/api/app/views/webui/groups/show.html.haml

```diff
@@ -32,7 +32,11 @@
   .card-body
     .tab-content
       .tab-pane.show.active#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
-        %h3 Group Members
+        %h3
+          Group Members
+          - if feature_enabled?(:responsive_ux)
+            = link_to('#', id: 'add-group-member', data: { toggle: 'modal', target: '#add-group-user-modal' }) do
```

Comment:
Setting the title `'Add Member'` instead of an id on this link would make the changes to the feature specs unneeded. It would also improve usability a tiny bit on desktop.